tests/test_serialization.py

```diff
@@ -0,0 +1,167 @@
+from typing import Type
```

Comment:
That's weird - why is this file appearing in diff?

Reply:
I have no idea what sort of merge mess I made, but the latest merge seems to have dealt with it.